src/test/java/by/itechart/api/controller/UserControllerIT.java

```diff
@@ -124,6 +127,27 @@ void createUser() throws Exception {
                 .andExpect(jsonPath("$.phone", is("1231231231")));
     }
 
+    @Test
+    void shouldThrowEmailDuplicationExceptionWhenUserWithEmailExists() throws Exception {
+        var createUserOne = getCreateUserDTO();
+        var createUserTwo = getCreateUserDTO();
+        this.mockMvc.perform(post("/users")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(createUserOne))
+                .with(httpBasic(ADMIN_USERNAME, ADMIN_AND_USER_PASSWORD))
+                .with(csrf()))
+                .andExpect(status().isCreated());
```

Comment:
Just nitpicking: this can be done just by running SQL script to create a user in the database using `@Sql` annotation